net/core/filter.c

```diff
@@ -5849,6 +5855,278 @@ static const struct bpf_func_proto bpf_tcp_gen_syncookie_proto = {
 
 #endif /* CONFIG_INET */
 
+#ifdef CONFIG_XDP_LUA
+
+static inline void verify_and_lock(void) {
```

Comment:
Suggested change, by @lneto

```
static inline lua_State *xdplua_lock(void) {
       struct xdplua_create_work *lw;

       lw = this_cpu_ptr(&luaworks);
	if (!lw->inuse) {
		lw->inuse = true;
		spin_lock(&lw->lock);
	}

        return lw->L;
}
```

Reply:
please, take care about `{`.. please review the whole patch..